src/client/src/rt-platforms/platform.ts

```diff
@@ -1,15 +1,15 @@
-import { Browser, Finsemble, OpenFin, PlatformAdapter, Symphony } from './'
+import { Browser, Finsemble, OpenFin, Symphony } from './'
+import { waitForObject } from '../rt-util'
 
 const urlParams = new URLSearchParams(window.location.search)
 
 const isFinsemble = 'FSBL' in window
 const isOpenFin = 'fin' in window
 const isSymphony = urlParams.has('waitFor') && urlParams.get('waitFor') === 'SYMPHONY'
```

Comment:
We need to fix this line, it is a temp workaround @bhavesh-desai-scratch and I did...

Reply:
@fdore will these constants actually work? Don't they get resolved before the platform is resolved?
I think that is one of the reasons we modified the isSymphony constant, it sometimes didn't get set correctly.

Would it work if they go inside the `getPlatformAsync` function?

Reply:
The only thing getPlatformAsync does is wait for symphony to be registered on the window, but don't you know from the start (.i.e from the launch url) that you are running on symphony? (disclaimer, I haven't tested the symphony use case)